mangle.json

```diff
@@ -34,6 +34,7 @@
   "props": {
     "cname": 6,
     "props": {
+      "$flags": "f",
```

Comment:
I would opt to keep this to `flags` to encourage use of library authors for additional extensions that we don't want to support first-party

Reply:
I've been going back and forth on it. It pains me that this is 23b just for `lags` 👁

Reply:
Hmm, the community being able to use this without looking at this file kind off boats well for me but yeah, one opinion 😅 also the types won't ever reflect .f unless we start doing two different type files

Reply:
true about types too (though that makes me think we should probably already be doing internal VS external files given we have other minified properties)